tests/unit/aiplatform/test_pipeline_jobs.py

```diff
@@ -462,6 +461,8 @@ def test_run_call_pipeline_service_create(
         mock_load_yaml_and_json,
         sync,
     ):
+        import yaml
```

Comment:
I'm wondering is there a reason why the `import yaml` is inside each test function instead of being global? Also if it is inside each function do we need to wrap it in a try-except statement?

Reply:
Responded above to Jialu's comment on the reasoning.

It's arguable that tests are not intended to be used/depended-on so we don't require as much UX friendly boilerplate on these optional dependency imports.

Reply:
Ok, sounds good